src/main/resources/public/js/app.js

```diff
@@ -3,19 +3,22 @@ var gameModel;
 //This function will be called once the page is loaded.  It will get a new game model from the back end, and display it.
 $( document ).ready(function() {
 
-  $.getJSON("model", function( json ) {
-    displayGameState(json);
-    gameModel = json;
+    var audio = new Audio('audio/Music.mp3');
+    audio.play();
+
+  $.getjson("model", function( json ) {
```

Comment:
This is the same issue we talked about on discord. You're renaming `getJSON` to `getjson`, and it's throwing errors in the browser's console. You should check for JS errors before submitting a PR. Also the grid straight up doesn't work, which you probably should've noticed.

Something you're doing is lowercasing function calls. `displaygamestate` is not the same as `displayGameState`.